recipes/qt/6.x.x/conanfile.py

```diff
@@ -712,11 +712,13 @@ def source(self):
             if os.path.isfile(file):
                 os.remove(file)
 
-        # workaround QTBUG-94356
-        replace_in_file(self, os.path.join(self.source_folder, "qtbase", "cmake", "FindWrapSystemZLIB.cmake"), '"-lz"', 'ZLIB::ZLIB')
-        replace_in_file(self, os.path.join(self.source_folder, "qtbase", "configure.cmake"),
-            "set_property(TARGET ZLIB::ZLIB PROPERTY IMPORTED_GLOBAL TRUE)",
-            "")
+        # workaround https://bugreports.qt.io/browse/QTBUG-94356
+        if Version(self.version) < "6.8.0":
+            replace_in_file(self, os.path.join(self.source_folder, "qtbase", "cmake", "FindWrapSystemZLIB.cmake"), '"-lz"', "ZLIB::ZLIB")
+            replace_in_file(self, os.path.join(self.source_folder, "qtbase", "configure.cmake"),
+                "set_property(TARGET ZLIB::ZLIB PROPERTY IMPORTED_GLOBAL TRUE)",
+                "")
+            
         if Version(self.version) <= "6.4.0":
```

Comment:
Unrelated, but this patch is no longer relevant for any versions built by CCI and can be dropped.